include/defines.php

```diff
@@ -1,4 +1,5 @@
 <div class="magick-header">
+<div id="cli-examples"></div>

```

Comment:
Why was this added?

Reply:
Ah, this and the other one was missed from an earlier attempt I was making for marking pages that should get the cli treatment. I'll remove these.